pkg/dashboard/frontend/src/lib/utils/generate-architecture-data.ts

```diff
@@ -255,25 +255,6 @@ export function generateArchitectureData(data: WebSocketResponse): {
       address: wsAddress,
     })
 
-    edges.push(
-      ...Object.entries(ws.targets).map(([eventType, target]) => {
-        return {
-          id: `e-${ws.name}-${target}-${eventType}`,
-          source: node.id,
-          target,
-          animated: true,
-          markerEnd: {
-            type: MarkerType.ArrowClosed,
-          },
-          markerStart: {
-            type: MarkerType.ArrowClosed,
-            orient: 'auto-start-reverse',
-          },
-          label: eventType,
-        }
-      }),
-    )
```

Comment:
This just outright deletes the Triggering edge. Do we want to get rid of these and only display the manageme policy for the websockets?